aiogram/dispatcher/dispatcher.py

```diff
@@ -232,20 +232,11 @@ async def _listen_update(self, update: Update, **kwargs: Any) -> Any:
                 "installed not latest version of aiogram framework",
                 RuntimeWarning,
             )
-            raise SkipHandler
+            raise SkipHandler()
```

Comment:
Isn't it the same?

Reply:
The same

Reply:
Then why you've changed it? Just curious.